samples/nrf9160/location/src/main.c

```diff
@@ -34,41 +31,6 @@ static void antenna_configure(void)
 	}
 }
 
-#if defined(CONFIG_NRF_CLOUD_PGPS)
-K_SEM_DEFINE(time_update_finished, 0, 1);
-
-static void date_time_evt_handler(const struct date_time_evt *evt)
```

Comment:
I think the sample probably still has to wait for an event from date_time lib, because otherwise it will try to use A-GPS/P-GPS before time has been got from the network. This is unfortunate, but I don't think there's a way around this. Getting time will take a while after the device has connected to the LTE network.

Reply:
New version implemented with cleaned date time handling.